install/install.queries.php

```diff
@@ -1561,9 +1561,9 @@ function encryptFollowingDefuse($message, $ascii_key)
                     }
                     unlink($csrfp_file); // delete existing csrfp.config file
                     copy($csrfp_file_sample, $csrfp_file); // make a copy of csrfp.config.sample file
-                    $inputData['data'] = file_get_contents($csrfp_file);
-                    $newdata = str_replace('"CSRFP_TOKEN" => ""', '"CSRFP_TOKEN" => "' . bin2hex(openssl_random_pseudo_bytes(25)) . '"', $inputData['data']);
-                    $jsUrl = $data_sent['url_path'] . '/includes/libraries/csrfp/js/csrfprotector.js';
+                    $data = file_get_contents($csrfp_file);
+                    $newdata = str_replace('"CSRFP_TOKEN" => ""', '"CSRFP_TOKEN" => "' . bin2hex(openssl_random_pseudo_bytes(25)) . '"', $data);
+                    $jsUrl = $inputData['data']['url_path'] . '/includes/libraries/csrfp/js/csrfprotector.js';
```

Comment:
You can also use relative path using a single dot:
```php
$jsUrl = './includes/libraries/csrfp/js/csrfprotector.js';
```

Reply:
> You can also use relative path using a single dot:
> 
> ```
> $jsUrl = './includes/libraries/csrfp/js/csrfprotector.js';
> ```

This configuration also works fine!